homeassistant/components/system_bridge/__init__.py

```diff
@@ -98,26 +98,35 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     await coordinator.async_config_entry_first_refresh()
 
-    _LOGGER.debug("Data: %s", coordinator.data)
-
     try:
         # Wait for initial data
         async with async_timeout.timeout(30):
-            while coordinator.data is None or all(
-                getattr(coordinator.data, module) is None for module in MODULES
-            ):
+            while not coordinator.is_ready:
                 _LOGGER.debug(
-                    "Waiting for initial data from %s (%s): %s",
+                    "Waiting for initial data from %s (%s)",
                     entry.title,
                     entry.data[CONF_HOST],
-                    coordinator.data,
                 )
                 await asyncio.sleep(1)
     except asyncio.TimeoutError as exception:
         raise ConfigEntryNotReady(
             f"Timed out waiting for {entry.title} ({entry.data[CONF_HOST]})."
         ) from exception
 
+    _LOGGER.debug(
```

Comment:
Do we need this log? Can't we assume the coordinator is ready when it logs the initial data below?

Reply:
True